core/src/main/java/com/novoda/noplayer/Heart.java

```diff
@@ -8,19 +8,19 @@
     private static final long HEART_BEAT_FREQUENCY_IN_MILLIS = 500;
 
     private final Handler handler;
-    private final Heartbeat heartbeatAction;
     private final long heartbeatFrequency;
 
+    private Heartbeat heartbeatAction;
+
     private boolean beating;
 
-    public static Heart newInstance(Heartbeat onHeartbeat) {
+    public static Heart newInstance() {
```

Comment:
Needed to pass the heart through the constructor on both players so needed to invert the `Heartbeat` dependency to occur when bound.